ksqldb-rest-app/src/test/java/io/confluent/ksql/rest/server/resources/ServerMetadataResourceTest.java

```diff
@@ -110,26 +107,5 @@ public void shouldReturnServerClusterId() {
         equalTo(ServerClusterId.of(KAFKA_CLUSTER_ID, KSQL_SERVICE_ID))
     );
   }
-
-
-  @Test
-  public void shouldGetKafkaClusterIdWithTimeout()
```

Comment:
Why are these tests being removed?

Reply:
Because these tests the memoized version of this class. But it's not memoized any more